.github/workflows/update-config.yml

```diff
@@ -0,0 +1,97 @@
+name: Pull Request Action
+permissions: write-all
+on:
+  workflow_dispatch:
+
+env:
+  GH_TOKEN: ${{ secrets.UPDATE_CONFIG_PAT }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Install jq and yq
+        run: |
+          sudo apt-get -y install jq
+          sudo wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq
+          sudo chmod +x /usr/bin/yq
+      - name: Get UbiquiBot Token
+        uses: tibdex/github-app-token@v1.7.0
+        id: get_installation_token
+        with:
+          app_id: ${{ secrets.UBIQUITY_BOUNTY_BOT_APP_ID }}
+          private_key: ${{ secrets.UBIQUITY_BOUNTY_BOT_PRIVATE_KEY }}
```

Comment:
@rndquu won't this be an issue on forks?

Reply:
Turns out I was looking in the wrong repositories. 

- https://github.com/ubiquity/telegram-ubiquibot/issues/13

Reply:
> shouldn't this be an issue on forks?

This [workflow](https://github.com/ubiquity/ubiquibot/blob/974e8b98646de124c62afcbf556d71d437107245/.github/workflows/update-config.yml) is meant to be run only manually. 

Forked repos will have to set `UBIQUITY_BOUNTY_BOT_APP_ID` and `UBIQUITY_BOUNTY_BOT_PRIVATE_KEY` to their own values should they want to run the "Update config" workflow.